kale/codegen/generate_code.py

```diff
@@ -99,9 +100,10 @@ def get_args(parameters):
         - 'parameters_types': parameters types as list
         - 'parameters_values': parameters default values as list
     """
-    parameters_names = sorted(parameters.keys())
-    parameters_types = [arg[0] for arg in parameters.values()]
-    parameters_values = [arg[1] for arg in parameters.values()]
+    sorted_parameters = collections.OrderedDict(sorted(parameters.items()))
+    parameters_names = sorted(sorted_parameters.keys())
+    parameters_types = [arg[0] for arg in sorted_parameters.values()]
+    parameters_values = [arg[1] for arg in sorted_parameters.values()]
```

Comment:
nits:
```suggestion
    sorted_parameters = dict(sorted(parameters.items()))
    parameters_names = list(sorted_parameters.keys())
    parameters_types = [arg[0] for arg in sorted_parameters.values()]
    parameters_values = [arg[1] for arg in sorted_parameters.values()]
```

explanation:
1. Ordered dict isn't required, since it's for maintaining the insertion order (but we don't insert anything new)
   >Python OrderedDict is a dict subclass that maintains the items insertion order. When we iterate over an OrderedDict, items are returned in the order they were inserted. A regular dictionary doesn't track the insertion order.
2. keys are already sorted (because of `sorted(parameters.items())`)
3. In some other case, using `sorted()` when getting the keys and not when getting the values would result in errors. That is, because you wouldn't know if the order of keys would change when you sort them.
   Here it is not an issue, since `sorted_parameters` is already sorted by keys

After that change you can also remove the `collections` import statement.